src/components/NextProject/NextProject.vue

```diff
@@ -0,0 +1,133 @@
+<template lang="html">
+    <nuxt-link :to="to" class="next-project">
+        <wp-image :image="image" mode="fullbleed" />
+        <div class="meta">
+            <h4>Next Project</h4>
+            <div class="credits">
+                <split-text class="title" :text="title" element="h2" />
+                <p
+                    v-for="credit in formattedCredits"
+                    v-if="credits.length"
```

Comment:
Can you make this a `<ul><li>` list, semantically that is what it is, rather than a paragraph of text. 

Also, you'd want to test `formattedCredits.length` here. Also just to say it, you don't need a `key` unless it's a component you are looping over, but it doesn't hurt to have one.